tests/test_quantum_distance.py

```diff
@@ -1,21 +1,36 @@
 import unittest 
 import os 
 import sys 
-current_path = os.path.dirname(os.path.abspath(__file__))
-sys.path.append(current_path + "/../neasqc_wp61/models/quantum/beta/")
-from quantum_distance import QuantumDistance as qd
-from quantum_k_nearest_neighbours_bert import QuantumKNearestNeighbours_bert as qkn
+import argparse
+
 import numpy as np 
 from qiskit import QuantumCircuit
 from qiskit.quantum_info import Statevector
 
+# The two lines below will be removed when converting the library to a package.
+current_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(current_path + "/../neasqc_wp61/models/quantum/beta/")
+from quantum_distance import QuantumDistance as qd
+from utils import normalise_vector, pad_vector_with_zeros
+
+
 class TestQuantumDistance(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls)-> None:
+        """
+        Set up the class for testing.
+        """
+        np.random.seed(args.seed)
 
-    def test_state_preparation(self):
+    def test_proper_state_preparation(self)-> None:
```

Comment:
Super vague. What does a "proper state prep" mean? Pls make that explicit in the docstring and develop the name a bit